godot_rl/wrappers/ray_wrapper.py

```diff
@@ -26,13 +26,13 @@ def __init__(
         config=None,
     ) -> None:
         self._env = GodotEnv(
-            env_path=env_path,
+            env_path=config["env_path"],
```

Comment:
Can all the options in `__init__` such as `env_path=None` etc be removed now as they seem to be stored in config?

Reply:
I think we only need e.g. as used in rllib example:
```return RayVectorGodotEnv(config=env_config, port=port, seed=seed)```

Perhaps even `seed` and `port` can be removed, but maybe it's more clear that they are dynamic options this way (we don't normally use a fixed port and seed for multiple envs).

We would need to change the register env also to make sure tests aren't broken.

I will check this later, try to remove them and see if there are any issues.